t.py

```diff
@@ -1,19 +1,23 @@
-from multiprocessing.dummy import Pool
-import re
-import shutil
-import traceback
-import pyperclip
+from __future__ import annotations
+
 import abc
-from curses.ascii import isspace
 import os
 import random
+import re
+import shutil
 import subprocess
-from tqdm import tqdm
 import sys
 import tempfile
-from typing import Generator, List, Literal, Tuple, Union
+import traceback
 import unittest
+from curses.ascii import isspace
+from multiprocessing.dummy import Pool
+from typing import Literal, Final
+
+import pyperclip
+from tqdm import tqdm
 
+# TODO: why rip here but not in a.py?
```

Comment:
Could you maybe explain if it was just forgotten or it's explicitly left out?

Reply:
I think it was just forgotten in `a.py`, but not entirely sure. It's kind of rare to explicitly test rip-relative addressing in instruction tests (it's only done in a few `xor` tests), so I think it doesn't matter if it's there or not, we can leave it out if you want to